api/api-reference.md

```diff
@@ -89,7 +89,7 @@ Create a new Tiger Postgres service. This is an asynchronous operation.
 ```json
 {
   "name": "test-2",
-  "service_type": "TIMESCALEDB",
+  "addons": "time-series",
```

Comment:
This change is good, because it's the _request_ body, not the _response_ body 👍. However, it should be an array:

```suggestion
  "addons": ["time-series"],
```